pom.xml

```diff
@@ -38,24 +38,21 @@
 
     <properties>
         <!-- Compile dependency versions (keep sorted alphabetically) -->
-        <ahc.version>2.12.1</ahc.version>
+        <ahc.version>${ahc-version}</ahc.version>
         <animal-sniffer.version>1.18</animal-sniffer.version>
         <antlr3.version>3.5.2</antlr3.version><!-- Spark, Stringtemplate and probably others -->
-	    <avro-ipc-version>1.10.0</avro-ipc-version>
-        <awssdk1.version>1.11.714</awssdk1.version>
+        <avro-ipc-version>1.10.0</avro-ipc-version>
         <awssdk1-swf-libs.version>1.11.22</awssdk1-swf-libs.version>
-        <awssdk2.version>2.14.3</awssdk2.version>
-        <bouncycastle.version>1.66</bouncycastle.version>
+        <bouncycastle.version>${bouncycastle-version}</bouncycastle.version>
         <camel.version>3.6.0</camel.version>
```

Comment:
I'd appreciate a comment on this line (and also in the parent declaration) that this needs to be kept in sync with the parent declaration.

Reply:
Ok - I'll follow up with another commit that adds a comment.